frontend/src/routes/FacilityBooking/ViewConflicts/ConflictBookingModal.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { ConfirmationModal } from '../../../components/ConfirmationModal'
+import { handleCreateBooking } from '../../../store/facilityBooking/action'
+import { Booking } from '../../../store/facilityBooking/types'
+
+export default function ConflictBookingModal(props: { booking: Booking | undefined }) {
+  const [open, setOpen] = useState(false)
```

Comment:
use better name, like modalIsOpen, setModalIsOpen; include state! 
```suggestion
  const [modalIsOpen, setmodalIsOpen] = useState<boolean>(false)
```

Reply:
Resolved in commit 509ab25176f49a4a836ede036eb3559e1af54833.